Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticSystem.cs

```diff
@@ -50,88 +34,12 @@ internal abstract class ElasticSystem<T>
         /// Query the elastic system for the current instantaneous value
         /// </summary>
         /// <returns>Current value of the elastic system</returns>
-        public T GetCurrentValue() => currentValue;
+        public abstract T GetCurrentValue();
```

Comment:
Removing the base implementation and making the base elastic system class completely abstract, cleaning up the base class interface and making the derived classes' behavior more explicit.